src/widgets/modals/ByProgramModal.js

```diff
@@ -73,7 +82,9 @@ const modalProps = ({ dispatch, program, orderType }) => ({
       const { requisitions } = getLocalizedStrings();
       const { maxOrdersPerPeriod, isEmergency } = orderType;
 
-      const requisitionsInPeriod = item.requisitionsForOrderType(program, orderType);
+      const requisitionsInPeriod = !customer
+        ? item.requisitionsForOrderType(program, orderType)
+        : item.customerRequisitionsForOrderTypeAndName(program, orderType, customer);
```

Comment:
This is for the properties for the modal: the row displays stats: how many requisitions so far in the period for example. So, it's changed to calculate correctly the number of requisitions in the period!